telemetry/aggregator_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func NewMock() *sql.DB {
-	db, err := sql.Open("postgres", "postgres://telemetry:newPassword@127.0.0.1:5432/telemetry_test")
+	db, err := sql.Open("postgres", "postgres://telemetry:newPassword@127.0.0.1:5432/telemetrydb?sslmode=disable")
```

Comment:
I added `sslmode=disable` as I did not want to mess with certs for the local DB setup and testing. But if that is something we care about, I can see what can be done for the container:)